_sass/blocks/explore/_filters.scss

```diff
@@ -41,3 +41,65 @@
 [aria-hidden=true] {
   display: none;
 }
+
+#unit-selector {
+  background: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.